mlir/lib/Conversion/SPIRVCommon/AttrToLLVMConverter.cpp

```diff
@@ -0,0 +1,61 @@
+//===- AttrToLLVMConverter.cpp - SPIR-V attributes conversion to LLVM -C++ ===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include <mlir/Conversion/SPIRVCommon/AttrToLLVMConverter.h>
+
+namespace {
+using namespace mlir;
```

Comment:
nit: I you do:
```c++
namespace mlir {
namespace {
```

you won't have to reopen namespace `mlir` in the middle of the file later on